go/vt/grpcclient/zstd.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package grpcclient
+
+import (
+	"io"
+
+	"github.com/klauspost/compress/zstd"
+)
+
+// ZstdCompressor is a gRPC compressor using zstd
+type ZstdCompressor struct{}
+
+// Name is "zstd"
```

Comment:
nit: this is self-evident from the code. i would instead say sometihng like "Name returns the name of the ZstdCompressor")

Reply:
or instead simply "Name is part of the encoding.Compressor interface."